sdk/storage/Azure.Storage.DataMovement/tests/StartTransferUploadTests.cs

```diff
@@ -1563,5 +1563,22 @@ public async Task StartTransfer_EnsureCompleted_Skipped()
             Assert.AreEqual(StorageTransferStatus.CompletedWithSkippedTransfers, transfer.TransferStatus);
         }
         #endregion
+
+        [Test]
+        public async Task SupportsLongFiles()
+        {
+            long fileSize = 5L * Constants.GB;
+
+            StorageResource srcResource = MockStorageResource.MakeSourceResource(fileSize, ProduceUriType.NoUri, maxChunkSize: Constants.GB);
+            StorageResource dstResource = MockStorageResource.MakeDestinationResource(ProduceUriType.ProducesUri, maxChunkSize: Constants.GB);
+            TransferManager transferManager = new TransferManager();
+
+            TransferOptions options = new();
+            TestEventsRaised events = new(options);
+            DataTransfer transfer = await transferManager.StartTransferAsync(srcResource, dstResource, options);
+            await transfer.AwaitCompletion();
+
+            Assert.IsEmpty(events.FailedEvents);
```

Comment:
Maybe also good to assert on `transfer.TransferStatus` to make sure its `Completed` (which indicates no failures).

Reply:
Along with that can this also be
`testEventsRaised.AssertSingleCompletedCheck();`
So in case if there was any failures in the pipeline in the future, they would be printed out.